cloudfront/iiif.wellcomecollection.org/terraform/cloudfront_distro/main.tf

```diff
@@ -1,100 +1,31 @@
 locals {
   subdomain_modifier = var.environment == "prod" ? "" : "-${var.environment}"
-
-  distro_alias = "iiif${local.subdomain_modifier}.wellcomecollection.org"
-
-  dds_domain   = "dds${local.subdomain_modifier}.dlcs.io"
-  iiif_domain  = "iiif${local.subdomain_modifier}.dlcs.io"
-  loris_domain = "iiif-origin.wellcomecollection.org"
-  dlcs_domain  = "dlcs.io"
+  distro_alias       = "iiif${local.subdomain_modifier}.wellcomecollection.org"
 }
 
 resource "aws_cloudfront_distribution" "iiif" {
   aliases = [
     local.distro_alias
   ]
 
-  origin {
-    domain_name = local.dds_domain
-    origin_id   = "dds"
-
-    custom_origin_config {
-      origin_protocol_policy = "match-viewer"
-      origin_ssl_protocols = [
-        "TLSv1",
-        "TLSv1.1",
-        "TLSv1.2",
-      ]
-
-      http_port  = 80
-      https_port = 443
-    }
-  }
-
-  origin {
-    domain_name = local.dlcs_domain
-    origin_id   = "dlcs"
-
-    custom_origin_config {
-      origin_protocol_policy = "match-viewer"
-      origin_ssl_protocols = [
-        "TLSv1",
-        "TLSv1.1",
-        "TLSv1.2",
-      ]
-
-      http_port  = 80
-      https_port = 443
-    }
-  }
-
-  origin {
-    domain_name = local.loris_domain
-    origin_id   = "loris"
-
-    custom_origin_config {
-      origin_protocol_policy = "match-viewer"
-      origin_ssl_protocols = [
-        "TLSv1.2",
-      ]
-
-      http_port  = 80
-      https_port = 443
-    }
-  }
-
-  origin {
-    domain_name = local.iiif_domain
-    origin_id   = "iiif"
-
-    custom_origin_config {
-      origin_protocol_policy = "match-viewer"
-      origin_ssl_protocols = [
-        "TLSv1",
-        "TLSv1.1",
-        "TLSv1.2",
-      ]
-
-      http_port  = 80
-      https_port = 443
-    }
-  }
-
-  origin {
-    domain_name = local.dlcs_domain
-    origin_id   = "dlcs_space_8"
-    origin_path = "/iiif-img/wellcome/8"
-
-    custom_origin_config {
-      origin_protocol_policy = "match-viewer"
-      origin_ssl_protocols = [
-        "TLSv1",
-        "TLSv1.1",
-        "TLSv1.2",
-      ]
-
-      http_port  = 80
-      https_port = 443
+  dynamic "origin" {
+    for_each = var.origins
+    content {
+      origin_id   = origin.value["origin_name"]
```

Comment:
minor: could we call this key `"origin_id"` for consistency?

Reply:
but then all the text doesn't line up nicely :(

👍 will fix this